.github/workflows/zip-templates-on-release.yml

```diff
@@ -0,0 +1,32 @@
+name: zip-templates-on-release
+on:
+  release:
+    types: [published]
+jobs:
+  zip-templates:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: npm install -g yarn
+      - run: yarn install
+      - run: sudo apt-get update && sudo apt-get install -y --allow-unauthenticated zip
+      - run: curl -s https://api.github.com/repos/ONSdigital/design-system/releases/latest | jq -r ".tarball_url" | wget -i - -O source.tar.gz
+      - run: mkdir design-system
+      - run: tar -xzf source.tar.gz -C design-system --strip-components=1
+      - run: cd design-system
+      - run: echo "GITHUB_RELEASE_TAG=$(curl -s https://api.github.com/repos/ONSdigital/design-system/releases/latest | jq -r ".tag_name")" >> $GITHUB_ENV
+      - run: RELEASE_VERSION=$GITHUB_RELEASE_TAG yarn cdn-bundle
+      - run: yarn install
+      - run: mkdir ../$GITHUB_RELEASE_TAG
+      - run: cd ../
+      - run: zip -r templates/templates.zip templates/*
+      - run: cp -R build/* ../$GITHUB_RELEASE_TAG
+      - uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: templates/templates.zip
+          tag: $GITHUB_RELEASE_TAG
+          overwrite: true
```

Comment:
I think it would be clearer as 3 jobs (one to install dependencies, one to build, another to zip and upload the templates zip). Each job can have multiple steps to break them up.

Reply:
Now I've got this working again and cut down the amount of commands needed, Im not sure its worth splitting it into 3 jobs. Because the way I would split it would mean the last 2 jobs would just be one command each. So what I've done instead is split it down into named steps